HED-schema-reduced.mediawiki

```diff
@@ -340,29 +350,55 @@ There are four ways to define the scope of an event.  The most common method is
 ** Urban
 ** Rural
 
-* Cognitive modifier <nowiki>{extensionAllowed}</nowiki>
-** Imagined <nowiki>[This is used to identity that the (sub)event only happened in the imagination of the participant, e.g.  imagined movements in motor imagery paradigms.]</nowiki>
-** Reflexive
-** Intentional
+* Experimental <nowiki>{extensionAllowed}</nowiki>
```

Comment:
Still not sure if this the right term. Action-related modifiers can be used for experimental or non-task related action for example.